posthog/hogql_queries/web_analytics/stats_table.py

```diff
@@ -27,6 +32,8 @@
 BREAKDOWN_NULL_DISPLAY = "(none)"
 
 
+# TODO: Extend `conversion_goal` support to queries besides `to_main_query`
```

Comment:
I think this isn't a to-do any more, we only need to support the main query.

The other queries are to show bounce rate, scroll depth, etc, but we don't need to do that with a conversion goal active